__frzr-deploy

```diff
@@ -341,7 +341,19 @@ main() {
 		INTEL_UCODE="initrd /${NAME}/intel-ucode.img"
 	fi
 
-	ADDITIONAL_ARGUMENTS=""
+	RESUME_ARGS=""
+	if [ -f "/home/swapfile" ]; then
+		RESUME_OFFSET=$(btrfs inspect-internal map-swapfile -r /home/swapfile)
+		
+		RESUME_DISK=$(mount | grep "/home" | awk '{print $1}')
+		PART_UUID=$(blkid -o value -s UUID ${RESUME_DISK})
+
+		RESUME_ARGS="resume=UUID=${PART_UUID} resume_offset=${RESUME_OFFSET}"
+
+		echo "Add resume hook to boot boot parameters: ${RESUME_ARGS}"
```

Comment:
"boot" is repeated

Reply:
Thanks. However I think this PR will be superceeded by the frzr rework carried over by me and @ruineka so that the user can choose the size instead of reling on a service to create the file, manually resizing and having to update before having that working...

Reply:
Oh, this seems like a better solution since you don't need an additional partition.

Reply:
We can bump the default swap size to something more reasonable to support hibernate.

Reply:
Both have upsides and downsodes, we will need to discuss to pick the least poisonous poison